DataConnectors/SquadraTechnologies.SecRMM.json

```diff
@@ -0,0 +1,119 @@
+{
+  "id": "SquadraTechnologiesSecRMM",
+  "title": "Squadra Technologies secRMM",
+  "publisher": "Squadra Technologies",
+  "descriptionMarkdown": "Use the Squadra Technologies secRMM Data Connector to push USB removable storage security event data into Azure Sentinel Log Analytics.",
+  "graphQueries": [
+    {
```

Comment:
Please modify the graphQueries as follows: You can have the current graphQuery mentioned as a sample query. That's because we have code in the connector framework that uses specifically this to add more to and the custom graphQuery you've added breaks that and we don't see the rendering right.
   "graphQueries": [ 
        { 
            "metricName": "Total data received", 
            "legend": "secRMM_CL", 
            "baseQuery": "secRMM_CL" 
        } 
    ],

Reply:
Hi, OK, fixed